PROCESS.md

```diff
@@ -0,0 +1,117 @@
+   See [CONTRIBUTING.md](CONTRIBUTING.md) for an overview of commit signing 
+   requirements and a guide on submitting a pull request.
+6. The pull request is merged after being approved as documented in the 
+   [Required Approvals](#Required-Approvals) section below.
+
+**NOTE**: Software design is often an iterative process with development. While 
+a design proposal is required, it is understood that development of the 
+proposed feature may begin prior to opening a design proposal issue in GitHub. It 
+is however highly encouraged to propose the design early in the interest of 
+respecting the contributor's time and effort. This avoids getting too far into 
+development without community involvement, which may lead to redesign and 
+extensive rework before changes can be approved.
+
+## Required Approvals
+
+Modifications of the contents of the OpenColorIO repository are made on a
+collaborative basis. Anyone with a GitHub account may propose a modification via
+pull request and it will be considered by the project Committers.
+
+Pull requests must meet a minimum number of Committer approvals prior to being
+merged. Rather than having a hard rule for all PRs, the requirement is based on
+the complexity and risk of the proposed changes, factoring in the length of
+time the PR has been open to discussion. The following guidelines outline the
+project's established approval rules for merging:
+
+* All changes must receive a minimum of two approvals from Committers listed in 
+  [COMMITTERS.md](COMMITTERS.md) prior to being merged. GitHub branch 
+  protections are enabled to enforce minimum committer count and Committer 
+  status. A Committer approval is denoted in a PR with a green check mark. A 
+  gray check mark beside an approval means the approver does not count toward 
+  the minimum approval count, but such approvals are welcome and encouraged 
+  from the community.
+
+* Small changes (bug fixes, docs, tests, cleanups) may be approved by any two 
+  Committers (other than the author).
+
+* Large or significant changes must receive both approvals from Committers that 
+  are not affiliated with the author or their company.
+
+* All pull requests, except critical emergency fixes, must be posted for a 48 
+  hour minimum prior to merging, even with the required approvals. This gives 
+  everybody a chance to see it across time zones and schedules.
+
+* If approval does not happen within two weeks of opening a pull request and 
+  there are no raised objections or unresolved discussions happening about the 
+  feature, a Committer that is affiliated with the author may approve it. At 
+  some point, we have to assume that the people who know and care are 
+  monitoring the PRs and that an extended period without objections is really 
+  assent. Notice must be given in the pull request 48 hours before merging 
+  (as early as the 12th day of an open PR) with the intent to merge. This 
```

Comment:
Minor, but we sometimes make this notice more than 48 hours before the 14 days has elapsed.  To me, the longer notice seems like a good thing but the new rule seems to forbid it (i.e. seems to say it cannot happen before day 12).  Propose replacing with the following sentence:

Notice must be given in the pull request between 2 and 4 days in advance of the intended merge date.

Reply:
Changed